static-assets/components/cstudio-common/common-api.js

```diff
@@ -2566,16 +2566,24 @@ var nodeOpen = false,
                       success: function () {
                         var pasteCb = {
                           success: function (pasteResponse) {
+                            opCallBack.success();
                             var filePath = pasteResponse.status[0];
                             var extension = filePath.split('.')[filePath.split('.').length - 1];
 
                             var editCb = {
                               success: function (newItem) {
                                 refreshFn(parentItemTo.item, newItem.item);
-                                opCallBack.success();
                               },
                               failure: function (errorResponse) {
-                                opCallBack.failure(errorResponse);
```

Comment:
Same thing here, the failure callback belongs to content duplication, not the form's save error callback